docs/upgrade/automatic.md

```diff
@@ -187,3 +187,65 @@ When upgrading to Harvester **v1.4.x**, Longhorn Manager may crash if the `Evict
 To prevent the issue from occurring, ensure that the `EvictionRequested` flag is set to `false` before you start the upgrade process.
 
 :::
+
+## CVE-2025-1974: Re-enable RKE2 ingress-nginx Admission Webhooks
+
+If you have previously [disabled the RKE2 ingress-nginx admission webhook](https://harvesterhci.io/kb/2025/03/25/cve-2025-1974) due to [CVE-2025-1974](https://nvd.nist.gov/vuln/detail/CVE-2025-1974), you will need to re-enable it after upgrading to Harvester v1.5.0, v1.4.3 or later with the following steps:
+
+1. Confirm that Harvester is using nginx-ingress v1.12.1 or later.
+
+  ```sh
+  $ kubectl -n kube-system get po -l"app.kubernetes.io/name=rke2-ingress-nginx" -ojsonpath='{.items[].spec.containers[].image}'
+  rancher/nginx-ingress-controller:v1.12.1-hardened1
+  ```
+
+1. Use `kubectl -n kube-system edit helmchartconfig rke2-ingress-nginx` to **remove** the following configurations from the resource.
+
+   * `.spec.valuesContent.controller.admissionWebhooks.enabled: false`
+   * `.spec.valuesContent.controller.extraArgs.enable-annotation-validation: true`
+
+1. The following is an example of what the new `.spec.ValuesContent` configuration should look like.
+
+  ```yaml
+  apiVersion: helm.cattle.io/v1
+  kind: HelmChartConfig
+  metadata:
+    name: rke2-ingress-nginx
+    namespace: kube-system
+  spec:
+    valuesContent: |-
+      controller:
+        admissionWebhooks:
+          port: 8444
+        extraArgs:
+          default-ssl-certificate: cattle-system/tls-rancher-internal
+        config:
+          proxy-body-size: "0"
+          proxy-request-buffering: "off"
+        publishService:
+          pathOverride: kube-system/ingress-expose
+  ```
+
+  :::info important
+  If the `HelmChartConfig` resource contains other custom ingress-nginx configuration, you must retain them when editing the resource.
+  :::
+
```

Comment:
An alternative shell script, it also fits the KB the make the patch easier

    ```sh
    kubectl get HelmChartConfig/rke2-ingress-nginx -n kube-system -o json | jq -r '.spec."valuesContent"' | yq e '.controller.admissionWebhooks.enabled = true' - > /tmp/updated-values.yaml

    cat <<'EOF' | kubectl apply -f -
    apiVersion: helm.cattle.io/v1
    kind: HelmChartConfig
    metadata:
      name: rke2-ingress-nginx
      namespace: kube-system
    spec:
      valuesContent: |-
    $(cat /tmp/updated-values.yaml | sed 's/^/    /')
    EOF
    ```

Reply:
Let's keep the doc similar to the KB for now, unless there are technical reasons why `kubectl edit` won't work. Since we told users to use `kubectl edit` in the KB (due to the way how the Harvester controller reconciles custom ingress TLS settings), we will use the same technique for the reverse.